alica_engine/src/engine/RunningPlan.cpp

```diff
@@ -825,6 +825,11 @@ const Configuration* RunningPlan::getConfiguration() const
     return _configuration;
 }
 
+IAlicaWorldModel* RunningPlan::getWorldModel() const
```

Comment:
RunningPlan has nothing to do with worldmodel. For BasicBehaviour and BasicPlan we can provide a getter function, for transitions and utility functions, summands, we can pass as argument (which would require codegen changes) or provide getters in base classes.

Reply:
in that case, I would suggest transitions should get a const reference to the world model. @abhi1287 I guess the utility functions & summands should also get a const reference?

Reply:
@veerajsk  yes